src/benchmark/operators/hash_join_benchmark.cpp

```diff
@@ -0,0 +1,757 @@
+
+#include <algorithm>
+#include <memory>
+
+#include "column_generator.hpp"
+#include "types.hpp"
+
+#include "../micro_benchmark_basic_fixture.hpp"
+#include "benchmark/benchmark.h"
+#include "concurrency/transaction_context.hpp"
+#include "expression/binary_predicate_expression.hpp"
+#include "expression/pqp_column_expression.hpp"
+#include "operators/join_hash.hpp"
+#include "operators/join_hash/multi_predicate_join_evaluator.hpp"
+#include "operators/print.hpp"
+#include "operators/table_scan.hpp"
+#include "operators/table_wrapper.hpp"
+#include "operators/validate.hpp"
+#include "storage/segment_accessor.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/value_segment.hpp"
+#include "type_cast.hpp"
+
+namespace {
+
+const size_t CHUNK_SIZE = 10000;
```

Comment:
1.) use thousands separator consistently.
2.) why 10k over 100k?
3.) the term `scale_factor` is not what you appear to be doing is. It's more of a row count, isn't it?

Reply:
We will discuss on Thursday what we will do with the benchmarks.

Reply:
@Bouncner `SCALE_FACTOR` is the number of rows of the smaller table, we changed the name to avoid confusion.